.github/workflows/agent-release.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     tags:
       - '*'
+  pull_request:
```

Comment:
This is going to publish a Python package on every push, no?

Reply:
Yes, this is just for testing.